homeassistant/components/v2c/sensor.py

```diff
@@ -75,6 +75,21 @@ class V2CSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         value_fn=lambda evse_data: evse_data.fv_power,
     ),
+    V2CSensorEntityDescription(
+        key="slave_error",
+        translation_key="slave_error",
+        value_fn=lambda evse_data: evse_data.slave_error.name,
+        entity_registry_enabled_default=False,
+    ),
+    V2CSensorEntityDescription(
+        key="battery_power",
+        translation_key="battery_power",
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.POWER,
+        value_fn=lambda evse_data: evse_data.battery_power,
+        entity_registry_enabled_default=False,
+    ),
```

Comment:
Please add this in a separate PR

Reply:
battery_power sensor only ?

Reply:
Both, then we can merge the fix of the intergation in a hotfix and add these 2 sensors as a new feature next release

Reply:
already split the version bump